agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp

```diff
@@ -213,6 +213,11 @@ bool monitor_thread::init()
 
 void monitor_thread::after_select(bool timeout)
 {
+    if (!logger_thread_name_set) {
```

Comment:
`logger_thread_name_set` - name?
Also should probably be moved to `before_select()`

Reply:
Oops! While I wrote it I set the thread on the logger but then I removed it and forgot to change it. I will rename this flag. I also forgot "m_" prefix.
The monitor thread does not override `before_select()`. It is redundant to add it just for that.

Reply:
It's perfectly fine to add it just for that.
You want to set apply this configuration as soon as possible.